test/fsync.test.js

```diff
@@ -21,21 +21,21 @@ test('fsync with sync', (t) => {
   const fd = fs.openSync(dest, 'w')
   const stream = new SonicBoom({ fd, sync: true, fsync: true })
 
-  t.ok(stream.write('hello world\n'))
-  t.ok(stream.write('something else\n'))
+  t.assert.ok(stream.write('hello world\n'))
+  t.assert.ok(stream.write('something else\n'))
 
   stream.end()
 
   const data = fs.readFileSync(dest, 'utf8')
-  t.equal(data, 'hello world\nsomething else\n')
+  t.assert.strictEqual(data, 'hello world\nsomething else\n')
 })
 
-test('fsync with async', (t) => {
-  t.plan(7)
+test('fsync with async', async (t) => {
+  t.plan(6)
```

Comment:
Plan now is six because using `readFileSync`, if the method fails, it will throw the error.